build.wake

```diff
@@ -103,10 +103,25 @@ def all variant =
 
     Pass (flatten x)
 
+def buildWakeLogViewer Unit =
+    require Some path = getenv "PATH"
+    else failWithError "Could not find cargo on path"
+
+    require Pass log_viewer =
+        makeExecPlan (whichIn path "cargo", "build", "--release",) Nil
```

Comment:
Its quite annoying depending on PATH here like this but I'm not sure what else to do right now. Ideally we'd download cargo within wake as part of the build I suppose? That might alleviate the docker head aches I'm having right now anyway.